cgi-bin/LJ/User/Administration.pm

```diff
@@ -238,18 +238,13 @@ sub banned_userids {
 
 sub ban_note {
     my ( $u, $ban_u, $text ) = @_;
-    my @banned;
 
-    if ( ref $ban_u eq 'ARRAY' ) {
```

Comment:
Make this function take user objects or ids, because there are callers that do both.